### DISCUSSION TITLE
zoom before move

### DISCUSSION THREAD

**Discussion by mrQuidome** [liodali/osm_flutter#305] [Q&A]
I have a goto current location button and I want to combine the move and the zoom. I have the following code:

      onPressed: () async {
        await controller.currentLocation();
        await controller.zoomIn();
      },

When I do this the zoom comes before the move. Is there a possibility to have it the other wat around?

**Comment by liodali** [comment]
reverse zoomIn before currentLocation

**Comment by mrQuidome** [comment]
Doesn't work at my side, reversing the line has no effect.

I want:
- move to the location first and then zoom in 

I got:
- zoom in first and then move to the new location

the locations are far apart, the move actions takes a lot more time to complete.

**Reply by mrQuidome** [reply]
No worries on this one, main goal for me is to understand the library. And I don't understand why this is happening :)

**Reply by liodali** [reply]
ah my bad i through that you want to zoom before 
for now that should be done inside currentLocation I have plan for that already 
i will added soon
